env0/data_project.go

```diff
@@ -30,9 +31,16 @@ func dataProject() *schema.Resource {
 				Computed:     true,
 			},
 			"parent_project_name": {
-				Type:        schema.TypeString,
-				Description: "the name of the parent project. Can be used as a filter when there are multiple subprojects with the same name under different parent projects",
-				Optional:    true,
+				Type:          schema.TypeString,
+				Description:   "the name of the parent project. Can be used as a filter when there are multiple subprojects with the same name under different parent projects",
+				Optional:      true,
+				ConflictsWith: []string{"parent_project_path"},
+			},
+			"parent_project_path": {
+				Type:          schema.TypeString,
+				Description:   "a path of ancestors projects divided by the prefix '|'. Can be used as a filter when there are multiple subprojects with the same name under different parent projects. For example: 'App|Dev|us-east-1' will search for a project with the hierarchy 'App -> Dev -> us-east-1' ('us-east-1' being the parent)",
+				Optional:      true,
+				ConflictsWith: []string{"parent_project_name"},
```

Comment:
does it not also conflict with the parent_project_id?

Reply:
you're correct...
I think I did not do that because the parent_project_id is also a computed field, and I was worried it will cause issues.
But looks like the SDK knows how to handle that use-case, so I'll push the change. (Sometimes the Terraform SDK is buggy).